examples/Programmable Buttons/Programmed External Keys.ino

```diff
@@ -0,0 +1,68 @@
+/*
+  Programmed External Keys (Keyboard)
+  
+  For the Arduino Leonardo/any other arduino board with usb interface
+  
+  Reads a key from push button & acts as a CTRL+C (Copy).
+  Reads another key from push button & acts as a CTRL+V (Paste).
+  
+
+  The circuit:
+  
+  The push button has 2 connectors. 
+  We must connect gnd and the second will be connected to any I/O pin. 
+
+connect register between (button & gnd)& then set pinMode() as INPUT.
+
+  created 18 March 2020
+  
+  by Prathamesh Sahasrabhojane (TheShubham99)
+
+*/
+
+#include "Keyboard.h"
+
+const int btnPin[] = {2, 3, 4, 5};
```

Comment:
@TheShubham99 this one still isn't resolved. Because of this, your sketch currently doesn't compile:
```
ProgrammableButtons:21:22: error: array bound is not an integer constant before ']' token

 int btnState[pincount];

                      ^
```

Reply:
Resolved